codestodians/komlie-seller
# Create store

#### What does this PR do?
Allows the seller to create a store

#### Description of Task to be completed?
Rework name on navbar
Account for page refresh with id and crypto js
Allow seller to create a store
Notification after store creation
Minor cleanup

#### How should this be manually tested?
Signup a new user
it should redirect to the create store page
Fill relevant details
Click submit
Ensure your env var is set
You should get a notification that the store was created
Error message on Login/Signup

#### Any background context you want to provide?
I put a next button because creating a store would be a two step process, the next step would be adding account details and any other necessary info

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/154254482

#### Screenshots (if appropriate)
![Screenshot 2019-06-19 at 12 32 17 PM](https://user-images.githubusercontent.com/20999363/59762094-4e689980-928e-11e9-9e2e-9df3c28d9626.png)


#### Questions:
Taking qustions
